uaclient/api/u/pro/security/vulnerabilities/cve/v1.py

```diff
@@ -0,0 +1,383 @@
+            doc="The ubuntu priority for the CVE",
+        ),
+        Field(
+            "notes",
+            data_list(StringDataValue),
+            False,
+            doc="A list of notes for the CVE",
+        ),
+        Field(
+            "affected_packages",
+            data_list(CVEAffectedPackage),
+            doc="A list of affected packages for this CVE",
+        ),
+        Field(
+            "fixable",
+            StringDataValue,
+            doc="The fixable status of the CVE",
+        ),
+        Field(
+            "cvss_score",
+            FloatDataValue,
+            False,
+            doc="The CVE cvss score",
+        ),
+        Field(
+            "cvss_severity",
+            StringDataValue,
+            False,
+            doc="The CVE cvss severity",
+        ),
+    ]
+
+    def __init__(
+        self,
+        *,
+        name: str,
+        description: str,
+        published_at: datetime.datetime,
+        ubuntu_priority: str,
+        fixable: str,
+        notes: Optional[List[str]] = None,
+        affected_packages: List[CVEAffectedPackage],
+        cvss_score: Optional[float] = None,
+        cvss_severity: Optional[str] = None
+    ):
+        self.name = name
+        self.description = description
+        self.published_at = published_at
+        self.ubuntu_priority = ubuntu_priority
+        self.fixable = fixable
+        self.notes = notes
+        self.affected_packages = affected_packages
+        self.cvss_score = cvss_score
+        self.cvss_severity = cvss_severity
+
+
+class CVEVulnerabilitiesResult(DataObject, AdditionalInfo):
+    fields = [
+        Field(
+            "cves",
+            data_list(CVEVulnerabilityResult),
+            doc="A list of CVEs that affect the system",
+        ),
+        Field(
+            "vulnerability_data_published_at",
+            DatetimeDataValue,
+            doc="The date the JSON vulnerability data was published at",
+        ),
+        Field(
+            "apt_updated_at",
+            DatetimeDataValue,
+            False,
+            doc="The date of the last apt update operation in the system",
+        ),
+    ]
+
+    def __init__(
+        self,
+        *,
+        cves: List[CVEVulnerabilityResult],
+        vulnerability_data_published_at: datetime.datetime,
+        apt_updated_at: Optional[datetime.datetime] = None
+    ):
+        self.cves = cves
+        self.vulnerability_data_published_at = vulnerability_data_published_at
+        self.apt_updated_at = apt_updated_at
+
+
+class CVEParser(VulnerabilityParser):
+    vulnerability_type = "cves"
+
+    def get_package_vulnerabilities(self, affected_pkg: Dict[str, Any]):
+        return affected_pkg.get(self.vulnerability_type, {})
+
+
+def vulnerabilities(
+    options: CVEVulnerabilitiesOptions,
+) -> CVEVulnerabilitiesResult:
+    return _vulnerabilities(options, UAConfig())
+
+
+def _vulnerabilities(
+    options: CVEVulnerabilitiesOptions,
```

Comment:
Let's throw an exception if unfixable and all are both true

Reply:
Fixed as well